source/technical/networks/networks_sdn.rst

```diff
@@ -27,7 +27,7 @@ Chameleon and OpenFlow
 
 BYOC is part of the expanded deployment for Chameleon's phase 2. It enables tenants to allocate OpenFlow switches controlled by their own OpenFlow controller. This capability is limited to the phase 2 hardware additions that include the Corsa DP2000 series OpenFlow switches and Skylake compute nodes. The Corsa switches are key to enabling the BYOC functionality.  These switches allow for the creation of mutually isolated forwarding contexts which can be thought of as virtual OpenFlow switches even though they are the native abstraction used by the Corsa DP2000s. Each isolated forwarding context can be configured to use its own OpenFlow controller. The Chameleon BYOC functionality enables tenants to specify the IP and port of an arbitrary OpenFlow controller when they create private networks.
 
-.. Important:: OpenFlow capabilities are site-dependent. At UC, both Haswell and Skylake node types can participate in an OpenFlow network; at TACC, only Skylakes are supported.
+.. Important:: OpenFlow capabilities are site-dependent. At UC and TACC, only Skylakes are supported.
```

Comment:
```suggestion
.. Important:: Currently (June 2021) only Skylake nodes can participate in OpenFlow experiments while we install new hardware as part of Chameleon Phase 3.
```

Reply:
There's a bit more context on this change. Over time many more nodes will be supported and we're in a weird transition period once the Haswells are gone.